<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QtabWidget{
border-radius:30px;
border-color:rgb(0, 0, 0)
}
*{
font-family:century gothic;
font-size:17px;
}
QLineEdit
{
border-radius:11px;
border-color:rgb(0, 0, 0);

}
QPushButton
{
background:rgb(139, 4, 35);
border-radius:13px;
color:rgb(255, 255, 255);
}
QPushButton:hover
{
background:rgb(216, 5, 37);
border-radius:13px;
color:rgb(255, 255, 255);
}

QToolButton{
background:rgb(139, 4, 35);
border-radius:15px;
}
QToolButton:hover{
background:rgb(216, 5, 37);
border-radius:15px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>1071</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="stackedWidgetPage1">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_ACTIVITE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>56</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Titre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date_activite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>localisation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>type_activite</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_ID">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>41</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_TITRE">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>91</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_DATE">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>141</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_LOCALISATION">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>191</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_TYPE">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>241</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouteractivite">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="ajouter_empl">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="affiche_emp">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier_emp">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer_emp">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2">
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/téléchargé (1)_preview_rev_1.png</normaloff>../../../Downloads/téléchargé (1)_preview_rev_1.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string> Recherche...</string>
      </property>
     </widget>
     <widget class="QTableView" name="table_activite">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>991</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/icon-pdf-white.png</normaloff>../../../Downloads/icon-pdf-white.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <zorder>lineEdit_12</zorder>
     <zorder>toolButton_2</zorder>
     <zorder>table_activite</zorder>
     <zorder>toolButton_3</zorder>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage3">
     <widget class="QPushButton" name="modifier_employer">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>310</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Localisation</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_spec_modif">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>191</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>56</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Titre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date_activite</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom_modif">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>141</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom_modif">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>91</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mail_modif">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>241</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID_ACTIVITE</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_modif">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>41</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>56</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mail</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage4">
     <widget class="QPushButton" name="supprimer_employe">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID Employé</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_supprimer_emp">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>131</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Downloads/téléchargé (1)_preview_rev_1.png</normaloff>../../../Downloads/téléchargé (1)_preview_rev_1.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="return_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Downloads/seta-small-direita-branca-300x258.png</normaloff>../../../Downloads/seta-small-direita-branca-300x258.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>27</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>AFFICHER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>supprimer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
